src/app/app.component.ts

```diff
@@ -6,5 +6,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.scss']
 })
 export class AppComponent {
-  title = 'xpanded-kingdom';
+  title: string = 'xpanded-kingdom';
```

Comment:
It is usually best practice to use public, private, etc accessors for component variables and methods.